README.md

```diff
@@ -58,6 +58,15 @@ This includes any file with `perl6` in the shebang, as well as any file
 with a `.p6`, `.pm6`, or `.pl6` extension. It also applies to any `.pm`,
 `.pl`, and `.t` files whose first line of code looks like Perl 6.
 
+Start the REPL with <kbd>M-x perl6-repl RET</kbd>. The following
```

Comment:
You changed it in the file to `run-perl6`, so you may want to update this as well.

Reply:
Good catch!